.github/workflows/pretest-and-test.yml

```
@@ -31,30 +31,18 @@ jobs:
     - name: Install
       run: |
         pip install -U pip wheel
-        pip install 'torch==${{ matrix.torch }}' --extra-index-url https://download.pytorch.org/whl/cpu
-        pip install -v -e .
+        pip install -e .[test] torch==${{ matrix.torch }} --extra-index-url https://download.pytorch.org/whl/cpu
         # Test PPE is importable with minimum dependency
-        python -c 'import pytorch_pfn_extras'
+        make import_check
```

(I think this is the matter of the preference) but I think most developers won't run this check locally. So I'd keep this as-is.

```suggestion
        python -c 'import pytorch_pfn_extras'
```